# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1521666147 -3600
#      Wed Mar 21 22:02:27 2018 +0100
# Branch stable
# Node ID 1fa6125c6d082fb168dcf52ecfc962a21888c084
# Parent  1a5ae7526c1ee0f40ef7f924423bc338a074004e
maint: fix typo in cset 1fbd8afcf625.

* test/local-functions/module.mk: rename "subfunction*" to "local_function*".

diff --git a/test/local-functions/module.mk b/test/local-functions/module.mk
--- a/test/local-functions/module.mk
+++ b/test/local-functions/module.mk
@@ -1,5 +1,5 @@
 local_functions_TEST_FILES = \
-  %reldir%/subfunctions.tst \
-  %reldir%/subfunction_eval.m
+  %reldir%/local_functions.tst \
+  %reldir%/local_function_eval.m
 
 TEST_FILES += $(local_functions_TEST_FILES)
